README.md

```diff
@@ -162,6 +130,33 @@ Other Heroku docs:
 - [Node.js on Heroku](https://devcenter.heroku.com/categories/nodejs)
 - [Best Practices for Node.js Development](https://devcenter.heroku.com/articles/node-best-practices)
 
+## Maintaining Researcher Portal
+### Whitelist database
+Researchers and teacher educators (TEs) looking to use data collected in Teacher Moments must be registered with the Teaching Systems Lab. To give researchers and TEs access to the researcher portal, their email addresses must be manually added to the `whitelist` database.
+
+### Access database
+Once a researcher or TE has been given access to the portal, they need to be given access to a specific instance of Teacher Moments. 
+
+For example, if they are using the Jessica Turner scenario for their thesis project, they might collect data using the following url: `https://teachermoments.teachingsystemslab.org/teachermoments/turner?KevinThesis20180319`. Teacher candidates would engage with the Teacher Moments simulation at that url and we would use that link to extract the data collected. To view the data, we would need to add an entry into the `access` database linking the TE's email to the url.
+
+### Consented_Email database
+All data collected by Teacher Moments is protected to ensure the privacy of participants. In order for a researcher or TE to view the data collected on a specific participant, the participant must fill out a [TSL consent form.](https://tsl.mit.edu/consent)
+
+To update Teacher Moments on who has consented to share their data, you need to:
+1. Download a CSV version of the consent spreadsheet. Search for `User Testing Consent Form (Responses)` in the TSL team Google Drive. Rename the file to `consented-latest-raw.csv` and save this file in the `threeflows/tmp` folder.
+
+2. Run the prep-consent script to process the CSV file using
+```
+$ yarn run prep-consent
+```
+This should generate 2 files: `consented-latest.json` and `consented-YYYY-MM-DD.json`.
+
+3. Run the update-consent script in heroku to update the live `consented_email` database using
+```
+$ cat ./tmp/consented-latest.json | heroku run --no-tty yarn run update-consent
+```
+
+
```

Comment:
Do you think it's worth adding a note here that these are sensitive files and you should delete the `tmp` folder after doing this operation?

Reply:
so the thing is, we also use the tmp folder to keep the credentials for the s3 and watson speech to text APIs. I haven't thought too much about security in this case since everything in the tmp folder is in the .gitignore file. 

Do you think we should add a note? I don't think it would hurt. But I also thought that the old script made a copy of the latest-consent.json with the date for record keeping. If we want people to delete the csv and json files after this operation, I should probably take out the `consented-YYYY-MM-DD.json` from the script.

Reply:
Maybe more targeted then, like `cat ./tmp/consented-latest.json | heroku run --no-tty yarn run update-consent | rm ./tmp/consented-latest.json`?

The change to remove the `consented-YYYY-MM-DD.json` file sounds good, that's leftover from how the private repo did this in the file system, and you've improved that to a better system in the database now!  So it'd be great to make these more temporary files that are just there while you're updating consent, and gone after.

re: security, the consent files are more sensitive than any of those other keys.  If we leaked one of the keys, we can rotate it immediately and then it's useless and can stay public forever with no ill effects.  If we leak people's email addresses, there's nothing we can do to recover from that being public.  So while access keys and credentials are very sensitive, the consent file is even more sensitive.

Reply:
that makes sense. Fixed that stuff!